graphql/resolvers.go

```diff
@@ -397,6 +414,7 @@ func (r *mutationResolver) SpawnHost(ctx context.Context, spawnHostInput *SpawnH
 		KeyName:              spawnHostInput.PublicKey.Key,
 		IsVirtualWorkstation: spawnHostInput.IsVirtualWorkStation,
 		NoExpiration:         spawnHostInput.NoExpiration,
+		TaskSync:             util.IsPtrSetToTrue(spawnHostInput.TaskSync),
```

Comment:
TaskSync relies on TaskID as well, correct? So I imagine this should be checked similar to UseProjectSetupScript.